sequencing/vcf_slice/dxapp.json

```diff
@@ -1,69 +1,90 @@
   "dxapi": "1.0.0",
-  "version": "0.0.1",
+  "version": "0.1.0",
   "categories": [
     "Variation Calling"
   ],
   "inputSpec": [
     {
       "name": "vcf_fn",
-      "label": "VCF Files",
+      "label": "VCF files",
+      "help": "Variant files to be sliced, in VCF format.",
       "class": "array:file",
       "optional": false,
-      "patterns" : ["*.vcf.gz"]
+      "patterns": ["*.vcf.gz"]
     },
     {
       "name": "vcfidx_fn",
-      "label": "VCF Index Files",
+      "label": "VCF index files",
+      "help": "Tabix index files corresponding to VCF files. Each VCF must have a corresponding index with matching base filename.",
       "class": "array:file",
       "optional": false,
-      "patterns" : ["*.vcf.gz.tbi"]
+      "patterns": ["*.vcf.gz.tbi"]
     },
     {
       "name": "region_fn",
       "label": "File of regions (chr:start-stop)",
+      "help": "Text file of genomic region coordinates. Variants within these ranges will be extracted from the given VCF files.",
       "class": "file",
-      "optional": false
+      "optional": false,
+      "patterns": ["*.txt", "*.regions", "*.list"]
     },
     {
-      "name": "suffix_fn",
-      "label": "Output File name suffix (Optional)",
+      "name": "suffix",
+      "label": "Output file name suffix (optional)",
+      "help": "(Optional) Include this given string in the output VCF filename extension, e.g. '*.sliced.SUFFIX.vcf.gz'.",
       "class": "string",
       "optional": true
     }
   ],
   "outputSpec": [
     {
       "name": "vcf_out",
-      "label": "Sliced VCF Files",
-      "class": "array:file"
+      "label": "Sliced VCF files",
+      "help": "Variant file containing only target regions, in VCF format.",
+      "class": "array:file",
+      "patterns": ["*.vcf.gz.tbi"]
     },
     {
       "name": "vcfidx_out",
-      "label": "Sliced VCF Index Files",
-      "class": "array:file"
+      "label": "Sliced VCF index files",
+      "help": "Tabix index file corresponding to the output VCF files.",
+      "class": "array:file",
+      "patterns": ["*.vcf.gz"]
     }
   ],
   "runSpec": {
-    "execDepends": [
-      {"name": "pigz"},
-      {"name": "openjdk-7-jre-headless"},
-      {"name": "tabix"},
-      {"name": "vcftools"}
-    ],  
     "interpreter": "bash",
-    "file": "src/vcf_slice.sh"
+    "file": "src/vcf_slice.sh",
+    "timeoutPolicy": {
+      "*": { "hours": 8 }
+    },
+    "release": "14.04",
+    "distribution": "Ubuntu"
   },
-  "access": {
-    "network": [
-      "*"
-    ]
+  "regionalOptions": {
+    "aws:us-east-1": {
+      "systemRequirements": {
+        "*": { "instanceType": "mem1_ssd1_x4" }
+      },
+      "assetDepends": [{"id": "record-FGpf7zQ0kfbZxj6q4ZzQ2BPG"}]
+    },
+    "azure:westus": {
+      "systemRequirements": {
+        "*": { "instanceType": "azure:mem2_ssd1_x4" }
+      },
+      "assetDepends": [{"id": "record-FGpfZ1j95qzXV43bP5y5JKgV"}]
+    }
```

Comment:
Please update record IDs to:
aws:us-east-1: record-FJ3zf8j08zVZFzqYG88Pbk7z
azure:westus: record-FJ4058j9Vf9ZBBVB2Q7b3VQz